test/distributed/test_c10d.py

```diff
@@ -360,16 +361,23 @@ def test_compare_set(self):
 
     def _create_client(self, addr, port, world_size):
         try:
-            client_store = dist.TCPStore(addr, port, world_size, timeout=timedelta(seconds=10))
+            client_store = dist.TCPStore(
+                addr, port, world_size, timeout=timedelta(seconds=10)
+            )
             self.assertEqual(b"value", client_store.get("key"))
             client_store.set("new_key", "new_value0")
             client_store.compare_set("new_key", "new_value1", "new_value0")
         except Exception:
-            logging.error('Caught exception: \n{}exiting process with exit code: {}'
-                          .format(traceback.format_exc(), MultiProcessTestCase.TEST_ERROR_EXIT_CODE))
+            logging.error(
+                "Caught exception: \n{}exiting process with exit code: {}".format(
+                    traceback.format_exc(), MultiProcessTestCase.TEST_ERROR_EXIT_CODE
+                )
+            )
```

Comment:
It's usually easier to review PRs if it doesn't contain formatting changes like these. Usually its better to send a separate PR with formatting changes.

Reply:
Yeah, I somehow messed it up. Let me revert the irrelevant changes.